test/vm.install

```diff
@@ -51,6 +52,11 @@ if [ "$PLATFORM_ID" = "platform:el10" ]; then
     printf  '[network]\nfirewall_driver = "nftables"\n' > /etc/containers/containers.conf
 fi
 
+if [ "$ID" = "arch" ]; then
+    # Disable PerSourcePenalties, our CI is too fast
+    echo "PerSourcePenalties no" >>/etc/ssh/sshd_config
```

Comment:
I suggest to write that into a separate file in /etc/ssh/sshd_config.d/ .

Reply:
Good idea!